crates/bevy_pbr/src/render/light.rs

```diff
@@ -560,15 +560,15 @@ pub fn extract_lights(
                 directional_light.shadow_projection.top
                     - directional_light.shadow_projection.bottom,
             );
-        let directional_light_texel_size =
-            largest_dimension / directional_light_shadow_map.size as f32;
+        let directional_light_texel_size = transform.radius_vec3a(Vec3A::ONE) * largest_dimension
```

Comment:
Shouldn't the largest dimension be calculated after the transform?
```rust
let directional_light_texel_size = transform.radius_vec3a(Vec3A::new(
  directional_light.shadow_projection.right - directional_light.shadow_projection.left,
  directional_light.shadow_projection.top - directional_light.shadow_projection.bottom,
  0.
)).abs().max_element() / directional_light_shadow_map.size as f32;
```

Reply:
Nice catch. Updated to use this approach, should give more accurate results.